agent/exec/containerd/adapter.go

```diff
@@ -71,6 +72,10 @@ type containerAdapter struct {
 	deleteResponse    *execution.DeleteResponse
 }
 
+func withNamespace(ctx context.Context) context.Context {
+	return namespaces.WithNamespace(ctx, "default")
```

Comment:
Is there value to using a swarm-specific namespace instead of the default namespace?

cc @stevvooe